include/zephyr/lw_sched/lw_sched.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SUBSYS_LIGHTWEIGHT_SCHEDULER
```

Comment:
Please use usual Zephyr header guard conventions:

```suggestion
#ifndef ZEPHYR_INCLUDE_LW_SCHED_LW_SCHED_H_
```

Reply:
Shall do.